client/view/Form/form.js

```diff
@@ -0,0 +1,82 @@
+import {
+  updateBrowsersStats,
+  updateGlobalCoverageBar,
+  updateToolsVersions,
+  hideStatsPlaceholder
+} from '../BrowserStats/browserStats.js'
+
+const API_HOST = 'http://localhost:5000/api';
+
+const form = document.getElementById('query_form');
+const textarea = document.getElementById('query_text_area');
+const errorMessage = document.getElementById('error_message');
+
+export function initForm() {
+  form.addEventListener('submit', async (e) => {
+    if (!form.checkValidity()) {
+      return;
+    }
+    const formData = new FormData(form);
+    const query = formData.get('query')
+    e.preventDefault();
+    form.classList.add('Form--justSend');
+    textarea.addEventListener('input', () => {
+      form.classList.remove('Form--justSend');
+    }, {
+      once: true,
+    })
+    updateStatsView(query);
+  })
+  textarea.addEventListener('keypress', (e) => {
+    if (e.keyCode === 13 && !e.shiftKey) {
+      e.preventDefault();
+      form.dispatchEvent(new Event('submit'));
+    }
+  })
+}
+
+export function renderError(message) {
+  errorMessage.innerHTML = message;
+  form.classList.add('Form--serverError');
+  textarea.addEventListener('input', () => {
+    form.classList.remove('Form--serverError');
+  }, {
+    once: true,
+  })
+}
+
+async function updateStatsView(query) {
+  let response;
+  try {
+    response = await fetch(`${API_HOST}?q=${encodeURIComponent(query)}`)
```

Comment:
Missing endpoint name

`${API_HOST}?/browsers?q`

Reply:
But... why does it work?...

Reply:
It looks like you haven't restarted the server for a long time. I tried running the `ariarzer` branch and got a 404 response